notebooks/notebooks/iclr2022/elo-vs-visits.ipynb

```diff
@@ -128,7 +128,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Elo curve again, but with updated match configs that are closer to the original KataGo configs
</pre><p>Not thrilled by the amount of code duplication between this and previous cell, but seems fine to address in another PR to avoid scope screep</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I'll just delete the previous cell since I've added this new cell's plot to the paper already.</p>